specification/containerservice/resource-manager/Microsoft.ContainerService/preview/2022-03-02-preview/managedClusters.json

```diff
@@ -5802,6 +5806,16 @@
       },
       "description": "Azure Key Vault key management service settings for the security profile."
     },
+    "ManagedClusterSecurityProfileWorkloadIdentity": {
+      "type": "object",
+      "properties": {
+        "enabled": {
+          "type": "boolean",
+          "description": "Whether to enable Workload Identity"
```

Comment:
Do we have a link we can include here that explains what workload identity _is_?

Reply:
We are working on for a public facing docs, will add it to the description once it's ready. cc @miwithro